### ISSUE TITLE
Download process does not start! Queue limited?


### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#369]
Occasionally a download process does not start, i.e. the download does not start directly, nor is it queued. Nothing happens.

Could it be that the queue has a limited capacity due to technical reasons?
In this case, the user should be informed of this and offered the option of setting a bookmark as an alternative.

**Comment by cemrich** [comment]
Downloads should always be queued and visible to the user, with some exceptions:
1. The device is not connected to any network (a toast message will be shown)
2. The required network type is not available (a toast message will be shown)

There are some constraints on how the download jobs are processed:
1. The required network type becomes unavailable during download (download will stay queued)
2. The device is low on storage (download will stay queued - I think...)

How many downloads are you triggering in parallel? Which system are you using?

**Comment by WebworkrNet** [comment]
**Device**: Samsung Galaxy A13 5G (SM-A136B/DSN)
**Operating System**: Android 13

There were already a lot of download jobs. I will count them when the phenomenon occurs again.

**Comment by WebworkrNet** [comment]
Today the phenomenon occurred while only 1 single download was actively downloading. Other downloads are "on hold" (without PUSH indicator, but with rotating download icon).

**Comment by WebworkrNet** [comment]
In the current case, there are 14 current downloads. As far as I remember, in other cases it was more than double.

I guess there's little free space as a cause. If so, the app should act in advance with a corresponding hint to the user instead of not reacting.
